### PR TITLE
feature/BNGP-3545-legal-agreement-parties-update-concov

### PR DESCRIPTION
**Pull request by soorajvdefra**

### CODE REVIEW

File: packages/webapp/src/routes/land/check-legal-agreement-file.js [hunk 1]
```diff
@@ -28,8 +29,9 @@ const handlers = {
       request.yar.set(constants.redisKeys.LEGAL_AGREEMENT_FILE_OPTION, 'no')
       return h.redirect(constants.routes.UPLOAD_LEGAL_AGREEMENT)
     } else if (checkLegalAgreement === 'yes') {
-      request.yar.set(constants.redisKeys.LEGAL_AGREEMENT_FILE_OPTION, 'yes')
-      return h.redirect(request.yar.get(constants.redisKeys.REFERER, true) || constants.routes.ADD_LEGAL_AGREEMENT_PARTIES)
+      if (legalAgreementType === 'conservation covenant') { return h.redirect(request.yar.get(constants.redisKeys.REFERER, true) || constants.routes.NEED_ADD_ALL_RESPONSIBLE_BODIES) } else {
```
Comment (KiranVarma02) [inline packages/webapp/src/routes/land/check-legal-agreement-file.js:32]: could we put string in const in some common place?

File: packages/webapp/src/routes/land/add-landowner-individual-conservation-covenant.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  post: async (request, h) => {
+    const individual = request.payload
+    individual.type = 'individual'
+    const { id } = request.query
+
+    const individualError = validateIndividual(individual)
+    if (!isEmpty(individualError)) {
+      return h.view(constants.views.ADD_LANDOWNER_INDIVIDUAL_CONSERVATION_COVENANT, {
+        individual,
+
```
Comment (AjinkyaJawalkar) [inline packages/webapp/src/routes/land/add-landowner-individual-conservation-covenant.js:51]: Remove unnecessary space

File: packages/webapp/src/routes/land/add-landowner-individual-conservation-covenant.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        err: Object.values(individualError),
+        firstNameError: individualError?.firstNameError,
+        lastNameError: individualError?.lastNameError
+
```
Comment (AjinkyaJawalkar) [inline packages/webapp/src/routes/land/add-landowner-individual-conservation-covenant.js:55]: Remove unnecessary space

**Review by AjinkyaJawalkar — COMMENTED** [review]
PR looks good to me. Need a quick PR approval from lead developer & QA checks to merge these changes

**Comment by AjinkyaJawalkar** [comment]
Please check the test coverage, It would be great if we get more coverage.

**Comment by teddmason** [comment]
@soorajvdefra what is the status of this PR, the ticket 3545 is set to done.  Can this pr be closed, has the change been made elsewhere?

**Pull request closed without merging**